source/otaJobParser/job_parser.c

```diff
@@ -224,6 +224,20 @@ static JSONStatus_t populateCommonFields( const char * jobDoc,
                                       &( result->fileId ) );
     }
 
+    if( jsonResult == JSONSuccess )
+    {    
+        buildIndexedFileQueryString( fileIndex,
+                                     "fileType",
+                                     8U,
+                                     queryString,
+                                     &queryStringLength );
+        jsonResult = searchUintValue( jobDoc,
+                                      jobDocLength,
+                                      queryString,
+                                      queryStringLength,
+                                      &( result->fileType ) );
```

Comment:
I do not think that this is standard across all the job documents.
For example, here is a job doc from a job that I recently created:
```
{
  "afr_ota": {
    "protocols": [
      "MQTT"
    ],
    "streamname": "AFR_OTA-some-stream-name",
    "files": [
      {
        "filepath": "NA",
        "filesize": 624244,
        "fileid": 0,
        "certfile": "sss:00213243",
        "update_data_url": null,
        "auth_scheme": null,
        "sig-sha256-ecdsa": "my-signature-goes-here"
      }
    ]
  }
}
```